grpc.go

```diff
@@ -668,6 +668,16 @@ func (a *agentGRPC) CreateContainer(ctx context.Context, req *pb.CreateContainer
 		return emptyResp, err
 	}
 
+	for _, e := range ociSpec.Process.Env {
+		if strings.Contains(e, "NVIDIA_VISIBLE_DEVICES") {
```

Comment:
then `foo=NVIDIA_VISIBLE_DEVICES` and  `vNVIDIA_VISIBLE_DEVICESar="foo" ` are valid? 
how about using `strings.HasPrefix(e, "NVIDIA_VISIBLE_DEVICES=")` ?

Reply:
yes, you are right. I'll fix it.